src/main/scala/core/seats/SeatStorage.scala

```diff
@@ -0,0 +1,39 @@
+package core.seats
+
+import core.Seat
+import utils.DatabaseConnector
+
+import scala.concurrent.{ExecutionContext, Future}
+
+sealed trait SeatStorage {
+  def getSeats: Future[Seq[Seat]]
+
+  def getSeat(id: Long): Future[Option[Seat]]
+
+  def takenSeats(screeningID: Long): Future[Seq[Seat]]
+
+  def saveSeats(newSeats: List[Seat]): Future[Seq[Seat]]
+
+}
+
+class DBSeatStorage(val databaseConnector: DatabaseConnector)(implicit executionContext: ExecutionContext)
```

Comment:
It is a good idea to have repository implementations in infrastructure package, not in domain package